sync/src/main/java/org/bf2/sync/ManagedKafkaSync.java

```diff
@@ -105,7 +111,8 @@ public void syncKafkaClusters() {
             if (existing == null) {
                 if (!remoteSpec.isDeleted()) {
                     reconcileAsync(ControlPlane.managedKafkaKey(remoteManagedKafka), localKey);
-                } else {
+                } else if (isConsistent(remoteManagedKafka)) {
```

Comment:
From the javadoc on `isConsistent` this condition makes sense but when reading the code it seems a bit strange to log a warning when something is consistent. I'm wondering if there is a way to rename `isConsistent` so that the usage reads better.

Reply:
This is a consistency check of local state.  The warning is logged because the remote state doesn't seem to match what we would expect.